packages/suite/src/actions/suite/storageActions.ts

```diff
@@ -313,7 +313,7 @@ export const removeDatabase = () => async (dispatch: Dispatch, getState: GetStat
     const rememberedDevices = getState().devices.filter(d => d.remember);
     // forget all remembered devices
     rememberedDevices.forEach(d => {
-        suiteActions.rememberDevice(d);
+        dispatch(suiteActions.forgetDevice(d));
     });
```

Comment:
@slowbackspace I was wondering if explicitly forgetting the devices is needed since we remove the whole DB anyway just a line below.

Reply:
These are really 2 things mixed together. Although we are are gonna remove the DB, we also need to update the devices inside reducers and set `device.remember` field to false. This is the field based on which we decide whether the device should be stored in idb.  Without this, in the UI it would still look like the device is remembered and there is even a change it would really got stored in DB again on some device change because of `remember` field still set to true.

Reply:
On the other hand.. it has never worked. Hm when I think about it, some time ago we were not dropping whole db, but instead we only cleared existing object stores. Back then this probably made more sense. I would try to remove it and do few tests to see if it works as supposed to. I am starting to lean to removing this (but not sure if we wanna do that now)